components/edit-collective/sections/HostVirtualCards.js

```diff
@@ -200,6 +209,9 @@ const HostVirtualCards = props => {
       message: <FormattedMessage id="Host.VirtualCards.Settings.Success" defaultMessage="Setting updated" />,
     });
   };
+  const handleEdit = card => () => {
+    setEditingVirtualCard(card);
+  };
 
```

Comment:
As per my other comment, maybe we could remove this function since it's only used at one place; 🤔 

```suggestion
```